packages/jaeger-ui/src/components/TracePage/SpanGraph/ViewingLayer.js

```diff
@@ -224,7 +225,18 @@ export default class ViewingLayer extends React.PureComponent<ViewingLayerProps,
   };
 
   /**
-   * Randers the difference between where the drag started and the current
+   * Resets the zoom to fully zoomed out.
+   *
+   * @param {Syntheticevent<HTMLButtonElement>} event - Click event created by clicking on button.
+   */
+  _resetTimeZoomClickHandler = (event: SyntheticEvent<HTMLButtonElement>) => {
+    event.stopPropagation();
```

Comment:
Are these (stopProp, preventDefault) still necessary if the button is on top of the SVG instead of under it?

Reply:
The handlers do not seem necessary. I am accustomed to being fairly restrictive about preventing event bubbling which may not necessary here. I have removed these two function calls since it behaves the same without them.